source/isaaclab_mimic/isaaclab_mimic/datagen/datagen_info.py

```diff
@@ -30,6 +31,7 @@ def __init__(
         eef_pose=None,
         object_poses=None,
         subtask_term_signals=None,
+        subtask_start_signals=None,
```

Comment:
Maybe this is coming from legacy code but we want type-hinting to live on function level and not docstrings. Please check here for style-guide: https://isaac-sim.github.io/IsaacLab/main/source/refs/contributing.html#type-hinting